include/i2c.h

```diff
@@ -52,7 +52,7 @@ extern "C" {
 #define I2C_SPEED_GET(cfg) 		(((cfg) & I2C_SPEED_MASK) \
 						>> I2C_SPEED_SHIFT)
 
-/** Use 10-bit addressing. */
+/** Use 10-bit addressing. DEPRECATED - Use I2C_MSG_ADDR_10_BITS instead. */
 #define I2C_ADDR_10_BITS		(1 << 0)
```

Comment:
`I2C_SLAVE_FLAGS_ADDR_10_BITS` references this macro in line 66 (I can't put the comment there). I guess `(1 << 0)` should be used there directly.